pkg/network/multi_network_policy.go

```diff
@@ -0,0 +1,37 @@
+package network
+
+import (
+	"os"
+	"path/filepath"
+
+	operv1 "github.com/openshift/api/operator/v1"
+	//"github.com/openshift/cluster-network-operator/pkg/names"
```

Comment:
if not used and commented I'd remove this

Reply:
fixed. thanks for the pointing out!